### PR TITLE
Support `_` arg

### LINKED ISSUE #19: append _ args to end, by default
I'm not sure if this is desirable behavior, but right now:

```
> dargs({'_': ['a', 'b']})
[ '--_=a', '--_=b' ]
>
```

Should it instead return the arguments without prepending the odd `--_=`?
Just request for comment before making a PR if people agree on the right behavior.
This also makes the behavior more symmetric with minimist.

### PR DESCRIPTION
**Pull request by chrisirhc**
Closes #19

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,5 +49,26 @@ module.exports = function (input, opts) {
 		}
 	});
 
+	if ('_' in input &&
+		!(Array.isArray(opts.excludes) && opts.excludes.indexOf('_') !== -1) &&
+		!(Array.isArray(opts.includes) && opts.includes.indexOf('_') === -1)) {
```
Comment (sindresorhus) [inline index.js]: Can you simplify this to not be double negative (You have both `!(` and `!==`)?

This check needs to be tested.

**Comment by sindresorhus** [comment]
Can you update the docs?

**Comment by chrisirhc** [comment]
Should be able to get to it by tomorrow.

File: index.js (hunk 1 above)
Comment (chrisirhc) [inline index.js]: Will do

**Comment by sindresorhus** [comment]
:cake:

**Comment by sindresorhus** [comment]
Closing for lack of activity. Happy to reopen if you ever get to this ;)

**Pull request closed without merging**